languages/csharp/BitwardenLibrary.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Runtime.InteropServices;
+
+namespace Bit.Sdk;
+
+internal static class BitwardenLibrary
+{
+    [DllImport("bitwarden_c", CallingConvention = CallingConvention.Cdecl)]
+    private static extern IntPtr init(string settings);
+
+    [DllImport("bitwarden_c", CallingConvention = CallingConvention.Cdecl)]
+    private static extern void free_mem(IntPtr clientPtr);
+
+    [DllImport("bitwarden_c", CallingConvention = CallingConvention.Cdecl)]
+    private static extern string run_command(string loginRequest, IntPtr clientPtr);
```

Comment:
One of the reasons to use SafeHandle is to get rid of the `IntPtr`.

```suggestion
    [DllImport("bitwarden_c", CallingConvention = CallingConvention.Cdecl)]
    private static extern BitwardenSafeHandle init(string settings);

    [DllImport("bitwarden_c", CallingConvention = CallingConvention.Cdecl)]
    private static extern void free_mem(BitwardenSafeHandle handle);

    [DllImport("bitwarden_c", CallingConvention = CallingConvention.Cdecl)]
    private static extern string run_command(string json, BitwardenSafeHandle handle);
```

Reply:
Changed